backend/openapi.yaml

```diff
@@ -2270,7 +2269,8 @@ components:
         business_id:
           description: Official business ID
           type: string
-          pattern: '^\d{7}-\d$'
+          nullable: true
+          pattern: '(^$|^\d{7}-\d$)'
```

Comment:
This would be much cleaner if the value is either a valid business id OR null. now it can be `null`, `""` or a valid id